.gitignore

```diff
@@ -39,6 +39,7 @@ hw/system/pad_control/rtl/pad_control_reg_top.sv
 
 # same for the C header file and linker scripts
 sw/device/lib/runtime/core_v_mini_mcu.h
+sw/device/lib/runtime/core_intr.h
```

Comment:
why?

Reply:
Good catch! This does not exist anymore. I will remove it and re-push